### PR TITLE
UISelect

### PR DESCRIPTION
**Pull request by igreulich**
* update UISelect component to allow a class to be added to the component container
* update the docs, added the live example

### CODE REVIEW

File: src/Components/UISelect.jsx [hunk 1]
```diff
@@ -208,54 +234,38 @@ module.exports = React.createClass({
...
-          className={elementClass}
-          onChange={this.updateSearch}
-          onKeyDown={this.onKeyDown}
-          placeholder={this.props.placeholder}
-          ref='searchInput'
-          tabIndex='-1'
-          type='text'
-          value={this.state.search}/>
+      showElement  = (
+        <input autoComplete='off' className={elementClass} onChange={this.updateSearch} onKeyDown={this.onKeyDown} placeholder={this.props.placeholder} ref='searchInput' tabIndex='-1' type='text' value={this.state.search}/>
```
Comment (mfunkie) [inline src/Components/UISelect.jsx]: Just curious, why did you remove it the tabbing here?

Comment (igreulich) [inline src/Components/UISelect.jsx]: Must have been in one of my reformatting macros that I use. I can put them back in, if need be.

Comment (mfunkie) [inline src/Components/UISelect.jsx]: Yeah, I'm picky and would like them back.  It improves readability in a component that needs it the most.

**Comment by igreulich** [comment]
Reformatted the ```input```.

**Comment by mfunkie** [comment]
Merge it then I'll pull into ObjectRestSpread
